cosmo/monitors/data_models.py

```diff
@@ -198,11 +198,10 @@ def get_new_data(self):
 
 class FUVDarkDataModel(BaseDataModel):
     files_source = FILES_SOURCE
-    subdir_pattern = '?????'
     cosmo_layout = False
 
-    # program_id = ['15771/', '15533/', '14940/', '14520/', '14436/', '13968/', '13521/', '13121/', '12716/', '12423/',
-    #               '11895/']
+    program_id = ['15771/', '15533/', '14940/', '14520/', '14436/', '13968/', '13521/', '13121/', '12716/', '12423/',
```

Comment:
Since this is a list, it might be better to name this as plural to avoid confusion when you iterate on it later

Reply:
good point